### PR TITLE
doc: update the ACRN on QEMU tutorial with latest kernel config

### PR DESCRIPTION
**Pull request by gvancuts**
Update the "Enable ACRN Over QEMU/KVM" tutorial:
* Remove the steps explaining how to add the Virtio blk driver
  to the Service VM kernel. It is now part of the default
  configuration
* Add a note to make it more obvious that the tutorial assumes
  that the compilation of ACRN and its kernel is done *inside*
  the QEMU VM that will serve as the Service VM for ACRN

Signed-off-by: Geoffroy Van Cutsem <geoffroy.vancutsem@intel.com>

### CODE REVIEW

File: doc/tutorials/acrn_on_qemu.rst [hunk 1]
```diff
@@ -149,38 +155,10 @@ Install ACRN Hypervisor
...
-      make
-
-#. Install the kernel and modules.
-
-   .. code-block:: none
-
-      sudo make modules_install
-      sudo cp arch/x86/boot/bzImage /boot
+#. Clone and configure the Service VM kernel repository following the instructions at
+   :ref:`build-and-install-ACRN-kernel` and using the `v2.5` tag. The User VM (L2 guest)
```
Comment (dbkinder) [inline doc/tutorials/acrn_on_qemu.rst:159]:
```suggestion
   :ref:`build-and-install-ACRN-kernel` and using the ``v2.5`` tag. The User VM (L2 guest)
```

File: doc/tutorials/acrn_on_qemu.rst [hunk 2]
```diff
@@ -149,38 +155,10 @@ Install ACRN Hypervisor
...
+   uses the ``virtio-blk`` driver to mount the rootfs, it is included in the default
```
Comment (dbkinder) [inline doc/tutorials/acrn_on_qemu.rst:160]:
```suggestion
   uses the ``virtio-blk`` driver to mount the rootfs. This driver is included in the default
```

File: doc/tutorials/acrn_on_qemu.rst [hunk 3]
```diff
@@ -149,38 +155,10 @@ Install ACRN Hypervisor
...
+   kernel configuration as of the `v2.5` tag.
```
Comment (dbkinder) [inline doc/tutorials/acrn_on_qemu.rst:161]:
```suggestion
   kernel configuration as of the ``v2.5`` tag.
```

**Review by dbkinder — CHANGES_REQUESTED** [review]
Thanks... a few tweaks..

File: doc/tutorials/acrn_on_qemu.rst (hunk 1 above)
Comment (gvancuts) [reply doc/tutorials/acrn_on_qemu.rst:159]: Ouch, this is bad ;-)

File: doc/tutorials/acrn_on_qemu.rst (hunk 2 above)
Comment (gvancuts) [reply doc/tutorials/acrn_on_qemu.rst:160]: OK

File: doc/tutorials/acrn_on_qemu.rst (hunk 3 above)
Comment (gvancuts) [reply doc/tutorials/acrn_on_qemu.rst:161]: Ouch, I did it again!

**Comment by gvancuts** [comment]
Updated the PR and force-pushed it. @dbkinder

**Review by dbkinder — APPROVED** [review]
LGTM, thanks.

**Pull request merged**